fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -62,6 +62,7 @@
 import java.util.Optional;
 
 public interface ConnectorMetadata {
+    String ENABLE_GET_STATS_FROM_METADATA = "enable_get_stats_from_metadata";
```

Comment:
we should not define the const in this high level api.

Reply:
Done